.travis.yml

```diff
@@ -43,9 +43,9 @@ jobs:
         - pip install --upgrade tox
       script:
         - tox -v --recreate
-    # backend tests (py3.6) start here
-    - name: 'Backend tests (py3.6)'
-      python: '3.6'
+    # backend tests (py3.7) start here
+    - name: 'Backend tests (py3.7)'
+      python: '3.7'
```

Comment:
This should be `3.7-dev`.

Reply:
https://docs.travis-ci.com/user/languages/python/

Reply:
> Recent Python branches require OpenSSL 1.0.2+. As this library is not available for Trusty, 3.7, 3.7-dev, 3.8-dev, and nightly do not work (or use outdated archive).

Reply:
https://github.com/travis-ci/travis-ci/issues/9831
```
language: python
python: 3.7
dist: xenial
sudo: true
```

`sudo: true` though... It's gonna be slow I think.

Reply:
Stick with `3.6` till full release then?

Reply:
I think so... We're waiting on Travis to implement something and it could take a while... /jk